data/yaml/venues_mapping.yaml

```diff
@@ -1,3 +1,5 @@
 ---
 iwclul:
-  venue: WS
+  venue: IWCLUL
+  joint:
+    2020.iwclul: WS
```

Comment:
Have you tested this? I believe the key needs to point to a volume, i.e., `2020.iwclul-1`.